pymc/distributions/discrete.py

```diff
@@ -878,7 +913,14 @@ class HyperGeometric(Discrete):
     rv_op = hypergeometric
 
     @classmethod
-    def dist(cls, N, k, n, *args, **kwargs):
+    def dist(
+        cls,
+        N: Optional[DISCRETE_DIST_PARAMETER_TYPES],
```

Comment:
I don't think this and the next two variables should be Optional. They are required by the distribution.